docgen/src/examples/tourism/App.js

```diff
@@ -234,11 +239,15 @@ const RoomType = RefinementList.connect(({items, refine, selectedItems}) => {
 });
 
 function Price() {
+  const theme = _.entries(Range.defaultTheme).reduce((acc, i) => {
```

Comment:
This is one way to do a more smart merge letting you have your own css redefining stuff and the default theme.